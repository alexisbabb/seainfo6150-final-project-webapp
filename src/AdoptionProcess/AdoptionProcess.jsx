import React from 'react'; 
import styles from "./AdoptionProcess.module.css";

const AdoptionProcess = (props) => {
    return (
        <article className = {styles.pageContent}>
            <header className = {styles.pageTitle} >
            Thank you for choosing adoption!
        
            </header>
            <div className = {styles.pageDetails}>
            <img className = {styles.pageImage} src="https://delawarehumane.org/wp-content/uploads/2019/03/34033908_10156259256476153_766746855991148544_o.jpg" alt="Adoption Process"/>
                <ol>
                    <li><u>Find your friend for life</u><br></br>
                    Our animals listings are updated in real time on our Adoptable Pets page. Animals have photos, pet profiles, current location and statuses updated regularly.</li>
                    <br></br>
                    <li><u>Submit your questionnaire online</u><br></br> 
                    Adoption Questionnaires submitted online are reviewed within 48hrs or immediately upon submission in-person. You can access the questionannaire from the individual pet pages.</li>
                    <br></br>
                    <li><u>Make an appointment or come in for a visit</u><br></br> 
                    Adoptions by appointment are from 9am-5pm on Monday through Thursday. No appointment needed adoptions can occur on open visitation days from 10am-5pm on Friday through Sunday. While we recommend submitting your questionnaire online before visiting on these designated days, we will have questionnaires available to submit in-person.</li>
                    <br></br>
                    <li><u>Meet your match</u><br></br>
                    Our Friend Makers will then introduce you to possible matches where you can spend time with them and ask any questions you have.</li>
                    <br></br>
                    <li><u>Take your new pet home</u><br></br>
                    Our Client Services staff will then go over necessary information and paperwork to complete your adoption! </li>
                </ol>
            </div>
        </article>
    )
}



export default AdoptionProcess