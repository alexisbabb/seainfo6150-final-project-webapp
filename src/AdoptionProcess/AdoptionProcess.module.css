.pageContent{
    background-image: linear-gradient(to bottom, rgba(255, 255, 0, 0.25), rgba(255, 0, 0, 0.25));
    font-family: 'Times New Roman', Times, serif; 
}

.pageTitle {
    font-size: 3em; 
    font-weight: bold;  
    text-align: center; 
    padding: 0.5em; 
    font-family: "Helvetica", sans-serif; 
}

.pageDetails{
    font-size: 1.25em;
    display: grid;
    grid-template-columns: 1fr;
    gap: 20px; 
    padding: 1em; 
}

.pageImage{
    width: 90%; 
    border-radius: 10%; 
    display: block; 
    margin-left: auto; 
    margin-right: auto; 
    margin-bottom: 1.5em; 
}

@media (min-width: 600px) {
    .pageDetails {
      grid-template-columns: repeat(2, 1fr);
    }
  }

