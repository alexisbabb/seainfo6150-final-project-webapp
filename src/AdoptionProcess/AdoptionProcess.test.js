import React from "react";
import AdoptionProcess from "./AdoptionProcess"

describe("AdoptionProcess tests", () => {
    it("renders correctly", () => {
        const src = "https://delawarehumane.org/wp-content/uploads/2019/03/34033908_10156259256476153_766746855991148544_o.jpg";
        const { container } = render(<AdoptionProcess src={src}/>);
        expect(container).toMatchSnapshot();
    });
});