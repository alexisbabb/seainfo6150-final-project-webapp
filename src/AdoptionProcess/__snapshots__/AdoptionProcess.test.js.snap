// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdoptionProcess tests renders correctly 1`] = `
<div>
  <article
    class="pageContent"
  >
    <header
      class="pageTitle"
    >
      Thank you for choosing adoption!
    </header>
    <div
      class="pageDetails"
    >
      <img
        alt="Adoption Process"
        class="pageImage"
        src="https://delawarehumane.org/wp-content/uploads/2019/03/34033908_10156259256476153_766746855991148544_o.jpg"
      />
      <ol>
        <li>
          <u>
            Find your friend for life
          </u>
          <br />
          Our animals listings are updated in real time on our Adoptable Pets page. Animals have photos, pet profiles, current location and statuses updated regularly.
        </li>
        <br />
        <li>
          <u>
            Submit your questionnaire online
          </u>
          <br />
          Adoption Questionnaires submitted online are reviewed within 48hrs or immediately upon submission in-person. You can access the questionannaire from the individual pet pages.
        </li>
        <br />
        <li>
          <u>
            Make an appointment or come in for a visit
          </u>
          <br />
          Adoptions by appointment are from 9am-5pm on Monday through Thursday. No appointment needed adoptions can occur on open visitation days from 10am-5pm on Friday through Sunday. While we recommend submitting your questionnaire online before visiting on these designated days, we will have questionnaires available to submit in-person.
        </li>
        <br />
        <li>
          <u>
            Meet your match
          </u>
          <br />
          Our Friend Makers will then introduce you to possible matches where you can spend time with them and ask any questions you have.
        </li>
        <br />
        <li>
          <u>
            Take your new pet home
          </u>
          <br />
          Our Client Services staff will then go over necessary information and paperwork to complete your adoption! 
        </li>
      </ol>
    </div>
  </article>
</div>
`;
